CHANGELOG.md

```diff
@@ -5,221 +5,316 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.11.0] - 2021-04-07
```

Comment:
Not sure what merits a move to 2.11.0 vs 2.10.1 -- Is this a big enough feature?

Reply:
2.10.1 would mean a patch, and since this is not fixing anything we shouldn't just increase the patch level.